aiida/tools/importexport/dbimport/utils.py

```diff
@@ -241,3 +243,48 @@ def deserialize_field(key, value, fields_info, import_unique_ids_mappings, forei
 
     # else
     return ('{}_id'.format(key), None)
+
+
+def start_header(archive, comment_mode, extras_mode_new, extras_mode_existing, debug):
+    """Print header for import"""
+    import os
+    from aiida.cmdline.utils import echo
+
+    archive = os.path.basename(archive)
+    title = 'IMPORT - !!! DEBUG MODE !!!' if debug else 'IMPORT'
+    result = '\n{}'.format(tabulate([['Archive', archive]], headers=[title, '']))
+
+    parameters = [
+        ['Comment mode', comment_mode],
```

Comment:
Why call it "rules" in export and "mode" in import?
Perhaps use "rules" everywhere?